Makefile

```diff
@@ -69,6 +67,18 @@ COLORIZE=$(SED) ''/PASS/s//$(COLOR_PASS)/'' | $(SED) ''/FAIL/s//$(COLOR_FAIL)/''
 DOCKER_NAMESPACE?=jaegertracing
 DOCKER_TAG?=latest
 
+# list for multi-arch image publishing
+TARGET_ARCHS ?= amd64 arm64
+
+BUILDX_PUSH ?= false
+DOCKER_BUILD ?= docker build --build-arg=TARGETARCH=$(GOARCH)
+ifeq ($(BUILDX_PUSH),true)
+	DOCKER_BUILD = DOCKER_CLI_EXPERIMENTAL="enabled" docker buildx build --builder=jaeger-builder --push $(foreach arch,${TARGET_ARCHS},--platform=linux/${arch})
+endif
+
+# make sure variables could pass to includes
```

Comment:
what is the meaning of this comment?

Reply:
make sure sub makefiles could use the make variables defined in root makefile.